src/ic-cdk-codegen/ic.did

```diff
@@ -0,0 +1,54 @@
+type canister_id = principal;
```

Comment:
Why do we need this file in crate root?
I saw the same file at `src/ic-cdk-macros/tests/ic.did` where it is used in a test.

Reply:
I think I just forgot to remove it.